src/Sarif.Viewer.VisualStudio/Sarif/Run.extensions.cs

```diff
@@ -98,5 +98,43 @@ public static string GetToolName(this Run run)
 
             return run.Tool.Driver.FullName ?? run.Tool.Driver.Name;
         }
+
+        public static bool HasNoResults(this Run run) => run.Results?.Count == 0;
```

Comment:
>Count [](start = 70, length = 5)

I love .net using signed values everywhere... so if .Count < 0 this returns true?

Reply:
Reversed the logic as you suggested in another comment, so the logic is now `run.Results?.Count > 0`.

---
In reply to: [523439091](https://github.com/microsoft/sarif-visualstudio-extension/pull/272#discussion_r523439091) [](ancestors = 523439091)